package.json

```diff
@@ -36,7 +36,7 @@
     "gulp-replace": "^1.1.3",
     "gulp-sass": "^5.1.0",
     "gulp-sourcemaps": "^3.0.0",
-    "gulp-svg-sprite": "^1.5.0",
+    "gulp-svg-sprite": "^2.0.1",
```

Comment:
@dhcole Why was this version changed? I get a conflicting peer dependency with `uswds-gulp` when this is bumped up, when I try to run `npm install`

Reply:
I believe it was an `npm audit` recommendation.

Reply:
Gotcha ok. I'm going to revert these packages back for now, as they're causing NPM and build errors. Makes sense to upgrade based on `npm audit` but perhaps best in a future task